server/webapp/WEB-INF/rails.new/webpack/views/dashboard/models/personalization_vm.js

```diff
@@ -98,6 +98,10 @@ function PersonalizationVM(currentView) {
 
     changeListeners.push(fn);
   };
+
+  this.isPipelineVisible = (pipeline) => {
```

Comment:
Again, the isPipelineVisible() probably does not belong here, but rather on the filter itself.